packages/react-native-reanimated/android/src/main/worklets/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+cmake_minimum_required(VERSION 3.8)
+
+file(GLOB_RECURSE WORKLETS_COMMON_SOURCES CONFIGURE_DEPENDS "${WORKLETS_COMMON_DIR}/*.cpp")
+
+# Consume shared libraries and headers from prefabs
+find_package(fbjni REQUIRED CONFIG)
+find_package(ReactAndroid REQUIRED CONFIG)
+
+if(${JS_RUNTIME} STREQUAL "hermes")
+    find_package(hermes-engine REQUIRED CONFIG)
+endif()
+
+add_library(
+    worklets
+    SHARED
+    ${WORKLETS_COMMON_SOURCES}
+    "${ANDROID_SRC_DIR}/main/cpp/AndroidLogger.cpp"
+)
+
+# includes
+target_include_directories(
+    worklets
+    PUBLIC
```

Comment:
```suggestion
    PRIVATE
```

Reply:
This is actually needed for Reanimated to be able to use Worklets.